Source/Scene/GlobeSurfaceTile.js

```diff
@@ -318,7 +318,7 @@ define([
 
         // If this tile's terrain and imagery are just upsampled from its parent, mark the tile as
         // upsampled only.  We won't refine a tile if its four children are upsampled only.
-        var isUpsampledOnly = defined(surfaceTile.terrainData) && surfaceTile.terrainData.wasCreatedByUpsampling();
+        var isUpsampledOnly = false;//defined(surfaceTile.terrainData) && surfaceTile.terrainData.wasCreatedByUpsampling();
```

Comment:
I misunderstood your comment and thought you were working on a better solution.
The idea of this Cesium fork is to add new opt-in capabilities to Cesium. But here the default behaviour is changed and it makes it impossible to compare gains.
I will add a simple mechanism and merge.